lib/tapioca/dsl/pipeline.rb

```diff
@@ -184,6 +188,16 @@ def report_error(error)
         handler.call(error)
         exit(1)
       end
+
+      sig { void }
+      def abort_if_pending_migrations!
+        return unless defined?(::Rake)
```

Comment:
I think you may not even need this one now that we're further down the path.

Reply:
I've removed this line as well 👍